### PR TITLE
Restart webservice tyr

### PR DESCRIPTION
**Pull request by Hoshiyo**
ref: https://github.com/CanalTP/navitia_deployment_conf/pull/218

### CODE REVIEW

File: fabfile/tasks.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def upgrade_all(up_tyr=True, up_confs=True, check_version=True, send_mail='no',
         execute(compare_version_candidate_installed)
     execute(kraken.swap_all_data_nav)
 
-    if env.use_load_balancer:
+    if env.use_load_balancer and env.name == 'prod':
```
Comment (pbougue) [inline fabfile/tasks.py]: Not a fan of 'is_prod' :)
It may be nicer to loop over the pools to be restarted.

File: fabfile/component/tyr.py [hunk 2]
```diff
@@ -682,6 +682,30 @@ def create_tyr_instance(instance):
...
+@task
+def reload_tyr_safe(server, safe=True):
+    """ Reload tyr on a specific server,
+        in a safe way if load balancers are available
+    """
+    safe = get_bool_from_cli(safe)
+    with settings(host_string=server):
+        if env.use_load_balancer and safe:
+            load_balancer.disable_node(server)
+        sudo("service apache2 reload")
```
Comment (pbougue) [inline fabfile/component/tyr.py]: some kind of `service apache2 graceful` would be even nicer

File: fabfile/tasks.py (hunk 1 above)
Comment (antoine-de) [inline fabfile/tasks.py]: yes `env.name == 'prod'` is forbidden :wink:

File: fabfile/component/tyr.py [hunk 3]
```diff
@@ -682,6 +682,30 @@ def create_tyr_instance(instance):
...
+@task
+def reload_tyr_safe(server, safe=True):
+    """ Reload tyr on a specific server,
+        in a safe way if load balancers are available
+    """
+    safe = get_bool_from_cli(safe)
+    with settings(host_string=server):
+        if env.use_load_balancer and safe:
+            load_balancer.disable_node(server)
+        sudo("apache2ctl graceful")
```
Comment (pbougue) [inline fabfile/component/tyr.py]: switch back to reload as it works everywhere

File: fabfile/env/platforms.py [hunk 4]
```diff
@@ -180,6 +180,11 @@ def make_ssh_url(serv, *args):
 
 env.stat_broker_exchange = 'stat_persistor_exchange'
 
+env.ws_hosts_1 = ''
```
Comment (pbougue) [inline fabfile/env/platforms.py]: Supposed to be a table, no?

**Pull request merged**